src/core/ext/transport/cronet/transport/cronet_status.h

```diff
@@ -0,0 +1,59 @@
+/*
+ *
+ * Copyright 2016 gRPC authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+
+#ifndef GRPC_CORE_EXT_TRANSPORT_CRONET_TRANSPORT_CRONET_STATUS_H
+#define GRPC_CORE_EXT_TRANSPORT_CRONET_TRANSPORT_CRONET_STATUS_H
+
+/*  HTTP/2 error codes are mapped to the following cronet net error codes */
```

Comment:
could you check if there is any similar implementation for Java?

Reply:
gRPC-Java has the similar implementation. The `Cronet error` will be passed to the application layer as a part of `gRPC error`. Here's the code: https://github.com/grpc/grpc-java/blob/master/cronet/src/main/java/io/grpc/cronet/CronetClientStream.java#L555-L561